website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -84,27 +83,31 @@ You can only do this with files ending in `auto.tfvars` or `terraform.tfvars`. Y
+Only members of a team with the "Manage variable set" permission can create, update, and delete variable sets. This permission is included for members of the [Owners Team](/terraform/cloud-docs/users-teams-organizations/permissions#organization-owners) or members of a team with the [Manage all projects](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-projects) or [Manage all workspaces](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-workspaces).
 
 HCP Terraform does not evaluate variable sets during Terraform runs for workspaces configured with `Local` [execution mode](/terraform/cloud-docs/workspaces/settings#execution-mode).
 
-To view variable sets for your organization, click **Settings**, then click **Variable sets**.
+Variable sets can be owned by an organization or by a project. To view variable sets, click **Settings** for your organization or project, then click **Variable sets**.
 
-The **Variable sets** page appears, listing all of the organization's variable sets. Click on a variable set to open it and review details about its variables and scoping.
+The **Variable sets** page lists all of the organization's or project's variable sets. Click on a variable set to open it and review details about its variables and scoping.
 
 ### Create Variable Sets
 
 To create a variable set:
 
-1. Go to the **Variable Sets** page for your organization and click **Create variable set**. The **Create a new variable set** page appears.
+1. Go to the **Variable Sets** page for your organization or project and click **Create variable set**. The **Create a new variable set** page appears.
 
 1. Choose a descriptive **Name** for the variable set. You can use any combination of numbers, letters, and characters.
 
 1. Write an optional **Description** that tells other users about the purpose of the variable set and what it contains.
 
 1. Choose a variable set scope:
-   - **Apply globally:** HCP Terraform will automatically apply this global variable set to all existing and future workspaces.
-   - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. This affects all current and future workspaces for any selected projects. After creation, users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace).
+   - Organization-owned
+      - **Apply globally:** HCP Terraform will automatically apply this global variable set to all existing and future workspaces.
+      - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. This affects all current and future workspaces for any selected projects. After creation, users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace).
```

Comment:
We try to avoid and/or where we can. 
```suggestion
      - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and projects to apply this variable set to. This affects all current and future workspaces for any selected projects. After creation, users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace).
```